modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectFieldLocalServiceTest.java

```diff
@@ -1393,6 +1406,273 @@ private String _testAddSystemObjectField(
 		return objectDefinitionName;
 	}
 
+	private void _testObjectFieldDefaultValue() throws Exception {
+		ObjectDefinition objectDefinition =
+			ObjectDefinitionTestUtil.addObjectDefinition(
+				_objectDefinitionLocalService,
+				Collections.singletonList(
+					ObjectFieldUtil.createObjectField(
+						ObjectFieldConstants.BUSINESS_TYPE_TEXT,
+						ObjectFieldConstants.DB_TYPE_STRING, true, false, null,
+						"able", "able", false)));
+
+		_objectDefinitionLocalService.publishCustomObjectDefinition(
+			TestPropsValues.getUserId(),
+			objectDefinition.getObjectDefinitionId());
+
+		// Add field with default value input as value
+
+		ObjectField objectField = _addCustomObjectField(
```

Comment:
@devpedrotavares when you take this code to the other method, change `objectFiled` to `picklistObjectField`.